botocore/docs/bcdoc/style.py

```diff
@@ -64,6 +64,15 @@ def underline(self, s):
     def italics(self, s):
         return s
 
+    def add_leading_space(self):
+        # Adds a leading white space if none exists.
```

Comment:
These writes can be any reStructuredText previously generated. This function pops whatever was written most recently and adds adds a space. We mainly want to prevent:
```
foo<code>bar</code>
``` 
from becoming 
```
foo`bar`
```
and instead want 
```
foo `bar`
```
since inline markup must be separated from surrounding text. I'll update function name and improve description.